### PR TITLE
Updated 01_HelloWorld Example

### PR DESCRIPTION
**Pull request by Pixmew**
Hello World example is modified for better appearence and more information in added ("print" Function) for better understanding.
It is now visually pleasing and gets a bit of extra information.

### CODE REVIEW

**Comment by Pixmew** [comment]
This is my first pull request ,am so nervous.It isn't much but i'm proud of it

File: examples/01_introduction/01_Hello world.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ fun main() {                        // 2
 
 1. Kotlin code is usually defined in packages. Package specification is optional: If you don't specify a package in a source file, its content goes to the default package.
 2. An entry point to a Kotlin application is the `main` function. In Kotlin 1.3, you can declare `main` without any parameters. The return type is not specified, which means that the function returns nothing. 
-3. `println` writes a line to the standard output. It is imported implicitly. Also note that semicolons are optional.
+3. `println` and `print` writes a line to the standard output with cursor ending no new line and in same line respectively. It is imported implicitly.
```
Comment (p7nov) [inline examples/01_introduction/01_Hello world.md:13]: In all the "Kotlin by Example" lessons we try to keep explanations as short as possible and explain only what happens in the code. For more detailed info, we offer the documentation.
This example shows how easy it is to write "Hello world" in Kotlin. I wouldn't overload it with details irrelevant for this exact line of code (no `print` function in it).

**Review by p7nov — COMMENTED** [review]
Hey! 
Thanks, it's always nice to hear from the community.
However, I have some concerns about this exact change, please see my comment below.

Regards, Pavel
Kotlin Technical Writer

**Comment by Pixmew** [comment]
So this PR is irrelevant as it overloads the user with unnecessary info am I right (I just thought it would help user to understand better ) what do think I should do .

**Comment by Pixmew** [comment]
So this PR is irrelevant as it overloads the user with unnecessary info am I right (I just thought it would help user to understand better ) what do think I should do .

**Comment by Pixmew** [comment]
And would you help me point me to right direction like what should I contribute next.?

**Comment by p7nov** [comment]
> So this PR is irrelevant as it overloads the user with unnecessary info am I right (I just thought it would help user to understand better ) what do think I should do .

I mean that all text in **Kotlin by example** explains what happens in the code. The code line is `println("Hello, World!") ` - no call of `print` in it, so the explanation of `print` seems irrelevant here. 

> And would you help me point me to right direction like what should I contribute next.?

It's awesome that you ask, thank you! 
There is a page [Contributing to Kotlin](https://kotlinlang.org/contribute.html) on kotlinlang.org. You are free to choose whichever way you like to help develop the Kotlin ecosystem.

**Comment by Pixmew** [comment]
is there any thing in particular in mind

**Comment by p7nov** [comment]
We usually don't ask people to do specific things. Just send us PRs or submit issues in [YouTrack](https://youtrack.jetbrains.com/issues/KT) if you feel that something needs an improvement, be it a doc page or a function in stdlib. You can also join the public Kotlin [Slack](http://kotlinlang.slack.com/) (get an invite [here](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up?_ga=2.134077326.1218289669.1605167163-154294388.1603171954) and discuss points of improvements with other Kotlin contributors.

**Comment by Pixmew** [comment]
ok Thankyou Sir

**Comment by p7nov** [comment]
Thank you!

**Pull request closed without merging**